tracing-subscriber/src/fmt/fmt_layer.rs

```diff
@@ -864,7 +866,9 @@ where
     /// the current span.
     ///
     /// [stored data]: ../registry/struct.SpanRef.html
-    pub fn scope(&self) -> Scope<'_, S>
+    #[deprecated(note = "wraps layer::Context::scope, which is deprecated")]
```

Comment:
nit: can we add the version here?
```suggestion
    #[deprecated(
        note = "wraps layer::Context::scope, which is deprecated",
        since = "0.2.19",
    )]
```